firmware/thruna/BMS/Src/App/states/App_AllStates.c

```diff
@@ -162,6 +163,10 @@ bool App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
         status = false;
         App_SharedStateMachine_SetNextState(state_machine, App_GetFaultState());
     }
+    else if (acc_warning)
+    {
+        // Cut torque requests to wheels??
```

Comment:
For now, warnings are only displayed to the driver for them to act on, if we can confirm that this algorithm does not produce false-positives, could potentially look into either cutting torque or upgrading to a fault.